docs/api.md

```diff
@@ -7,18 +7,112 @@ Loosely based off the example in the README
 
 | Name | Type | Description | Example |
 | ------- | ------- | ------- | ------- |
-| **[black_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
-| **date_played** | *date-time* |  | `"2015-01-01T12:00:00Z"` |
-| **game_record** | *string* |  | `"<raw sgf string>"` |
-| **game_server** | *string* |  | `"KGS"` |
-| **rated** | *boolean* |  | `true` |
-| **[white_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
+| **[black_id](#resource-player)** | *number* | black player's id | `42.0` |
+| **date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
+| **game_record** | *string* | game record in SGF | `"<raw sgf string>"` |
+| **game_server** | *string* | game server id | `"KGS"` |
+| **id** | *number* | this could be a uuid to make it db agnostic | `42.0` |
+| **rated** | *boolean* | whether this was a rated game (on the server?) | `true` |
+| **[white_id](#resource-player)** | *number* | white player's id | `42.0` |
+
+### Game Creation
+
+Report a game result
+
+```
+POST /api/v1/games
+```
+
+#### Required Parameters
+
+| Name | Type | Description | Example |
+| ------- | ------- | ------- | ------- |
+| **auth:black_token** | *uuid* | secret token for black player | `"01234567-89ab-cdef-0123-456789abcdef"` |
+| **auth:server_token** | *uuid* | secret server token | `"01234567-89ab-cdef-0123-456789abcdef"` |
+| **auth:white_token** | *uuid* | secret token for white player | `"01234567-89ab-cdef-0123-456789abcdef"` |
+| **game:black_id** | *number* | black player's id | `42.0` |
+| **game:date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
+| **game:game_record** | *string* | game record in SGF | `"<raw sgf string>"` |
+| **game:game_server** | *string* | game server id | `"KGS"` |
+| **game:rated** | *boolean* | whether this was a rated game (on the server?) | `true` |
+| **game:white_id** | *number* | white player's id | `42.0` |
+
+
+
+#### Curl Example
+
+```bash
+$ curl -n -X POST http://dev.usgo.org/api/v1/games \
+  -d '{
+  "game": {
+    "black_id": 42.0,
+    "white_id": 42.0,
+    "game_server": "KGS",
+    "rated": true,
+    "date_played": "2015-01-01T12:00:00Z",
+    "game_record": "<raw sgf string>"
+  },
+  "auth": {
```

Comment:
Are you planning on changing the methods? Currently these are specified as query params.